x-pack/packages/ml/field_stats_flyout/options_list_with_stats/option_list_with_stats.tsx

```diff
@@ -68,13 +73,13 @@ export const OptionListWithFieldStats: FC<OptionListWithFieldStatsProps> = ({
   const comboBoxOptions: DropDownLabel[] = useMemo(
     () =>
       Array.isArray(options)
-        ? options.map(({ isEmpty, hideTrigger: hideInspectButton, ...o }) => ({
+        ? options.map(({ isEmpty, ...o }) => ({
+            key: o.key,
```

Comment:
as `o` is spread below, I don't believe `key: o.key` is needed

Reply:
Updated here [`9b9ee5b` (#195485)](https://github.com/elastic/kibana/pull/195485/commits/9b9ee5b252977641fe1fa4b9550acbb855c48eed)